### DISCUSSION TITLE
BACKUP_FILENAME_EXPAND

### DISCUSSION THREAD

**Discussion by boulala** [offen/docker-volume-backup#221] [Q&A] [answered]
Hi Community
I hope someone can help me as i can't get the BACKUP_FILENAME_EXPAND Option to work for me.
I tried multiple approaches by setting the variable on the container with the volume, setting the variable on the backup container, setting the hostname parameter etc.
I'm using docker-compose and would like to specifi the variable name for each backup-file and this option seemed perfect for that.
Maybe someone can show me an example how this should be set.
Thanks

**Comment by m90** [comment, marked as answer]
This testcase has an example for how to use the feature https://github.com/offen/docker-volume-backup/blob/dc8172b67313efc8ae9962b8da7a661914ffa3e0/test/s3/docker-compose.yml

Maybe it'd be helpful if you could share the code that you are trying to use, but does not work, maybe it's a simple thing to fix?

**Reply by boulala** [reply]
Thanks for the link.
In the mean time I found the solution myself (same as in the link you shared) :)
